### PR TITLE
feat: added login page

### PR DESCRIPTION
**Pull request by Developers-codz**
Created login page 
login page link - https://deploy-preview-7--beadsandbeyonds.netlify.app/login.html
please do review and approve. Thank you :)

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                <h1 class="header-logo">Beads & Beyonds....</h1>
+                <small class="slogan">Decorate your world with us</small>
+    
+            </div>
+            <div class="mr-auto page-link-wrapper font3">
+                <ul class="style-list-none">
+                    <li class="inline-block"><a href="index.html" class="page-link decor-none">Home</a></li>
+                    <li class="inline-block"><a href="shop.html" class="page-link decor-none">Shop</a></li>
+                </ul>
+
```
Comment (arjun-computer-geek) [inline login.html:30]: remove extra spaces

File: styles/login.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+.input-field::placeholder {
+  font-weight: 500;
+  font-size: 1.2rem;
+}
+.input-field:focus {
+  outline: none;
+  border-bottom: 2px solid var(--primary);
+}
+.demo-login {
+  margin: 1.5rem 0rem 0rem 0rem;
```
Comment (arjun-computer-geek) [inline styles/login.css]: use margin-top instead margin here

File: login.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    <title>Login | Beads & Beyonds</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./styles/login.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="https://beadsui-dev.netlify.app/css/main.css">
+
+</head>
+<body>
+    <div id="body-wrapper">
+    <Header class="">
```
Comment (vrx29) [inline login.html]: H should be in lowercase in header tag.

**Review by vrx29 — APPROVED** [review]

File: styles/login.css (hunk 2 above)
Comment (Developers-codz) [reply styles/login.css]: done!

**Comment by arjun-computer-geek** [comment]
f
Screenshot_2022-02-12_13-30-40


**It will be better if  some padding in bottom**

**Review by arjun-computer-geek — APPROVED** [review]
else everything looking good to me

File: JS/login.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+var passInput = document.querySelector("#login-pass");
+var eyeBtn = document.querySelector("#eye-btn");
+
```
Comment (ParthParolekar) [inline JS/login.js]: use let/const instead of var

**Review by ParthParolekar — APPROVED** [review]
Everything else looks great!

File: styles/login.css [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  border: 2px solid transparent;
+  color: var(--primary);
+  font-size: 1rem;
+}
+.demo-login:hover {
+  transform: scale(1.03, 1.03);
+  border: 2px solid var(--primary);
+}
+.login-btn {
+  margin: 1.5rem 0rem 0rem 0rem;
```
Comment (kushanksriraj) [inline styles/login.css:49]: We can keep other values as just 0.

**Pull request merged**